### PR TITLE
Curatory reviews

### PR DESCRIPTION
**Pull request by hornmo**
- Remove allocation information from review request templates (#268 )
- Add missing CRUD actions for reviews
- Add action for retrieving reviews of a curatoryGroup (`/curatoryGroups/<id>/reviews`)
- Respect combo status for list of Identifier Reviews
- Add dedicated ReviewRequestService
- Add new domain class `AllocatedReviewGroup` representing n:m links between RR and CuratoryGroups
- Enable usage of stdDesc refdata for Requests
- Removed unneeded method restriction from Secured annotations
- Fix variable usage in TitleController (#183 )
- Add index for Combo status
- Respect title type in upsert (#273 )
- Filter out deleted title in non-sync upsert
- Remove default RR for titles without class one ids

### CODE REVIEW

File: server/gokbg3/grails-app/controllers/org/gokb/rest/ReviewRequestService.groovy [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          new AllocatedReviewGroup(group: it, review: req).save(flush:true,failOnError:true)
+        }
+      }
+    }
+
+    req
+  }
+
+  def allocateGroups(rr, user) {
+
```
Comment (philboeselager) [inline server/gokbg3/grails-app/controllers/org/gokb/rest/ReviewRequestService.groovy]: missing on purpose?

Comment (hornmo) [reply server/gokbg3/grails-app/controllers/org/gokb/rest/ReviewRequestService.groovy]: There are some missing parts here, I will re-request once everything is fixed.

File: server/gokbg3/grails-app/services/org/gokb/TitleLookupService.groovy [hunk 2]
```diff
@@ -556,13 +564,16 @@ class TitleLookupService {
...
 
-              if(added) {
-                ReviewRequest.raise(
-                    the_title,
-                    "'${metadata.title}' added as a variant of '${the_title.name}'.",
-                    "Title was matched via secondary id, but had a different name.",
-                    user, project
-                    )
+              if (added) {
+                def rr = reviewRequestService.raise(
```
Comment (philboeselager) [inline server/gokbg3/grails-app/services/org/gokb/TitleLookupService.groovy]: There's nothing further done with `rr` - so maybe drop that assignment?
(Same for the following cases.)

**Review by philboeselager — APPROVED** [review]
No blockers. Please check missing method content (see first comment) and maybe leave out the obsolete assignments (see second comment).

**Pull request merged**